book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :book
+  attr_reader :rentals
```

Comment:
- [ ] Dear @BiancaMe you did an amazing job and implemented the things with your knowledge for this milestone, you just need to keep a few things in mind which will help you make your console application run without any problem, and that is equally important for the sake of the requirement for this `association` milestone for your library application. As you can see you added the `rentals` with the `attr_reader` attribute, which means you can just read the data for the book rental because `attr_reader` create the getter method behind the scene in the ruby, just think about what you want to add the rentals later at some point, but your scope of the rental doesn't allow you to add the data into it, so that's why you should put `: rentals` alongside the `attr_accessor` which create not only setter but also getter for your newly added book to build the relationship correctly between rentals and books.